content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -0,0 +1,54 @@
+---
+content_type: "api_reference"
+title: Kubetlet Checkpoint API
+weight: 10
+---
+
+## Kubelet Checkpoint API
+
+Checkpointing a container is the functionality to create a stateful copy of a
+running container. The stateful copy of the container can be moved to another
+system and after restore the container will continue to run at exactly the same
+point it was checkpointed. The concept of checkpointing a container and
+restoring a container can be seen as pausing and unpausing a container with the
+possibility to transfer the container to another system between pausing and
+unpausing.
```

Comment:
OK, so now this is in place, I'll suggest wording tweaks:
```suggestion
Checkpointing a container is the functionality to create a stateful copy of a
running container. Once you have a stateful copy of a container, you could
move it to a different computer for debugging or similar purposes.

If you move the checkpointed container data to a computer that's able to restore
it, that restored container continues to run at exactly the same
point it was checkpointed. You can also inspect the saved data, provided that you
have suitable tools for doing so.
```

I don't want to give people the impression that restoring to a kubelet is supported - it isn't. And we [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in our docs.